NOTICE

```diff
@@ -3,3 +3,6 @@ Copyright 2017 The Apache Software Foundation.
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
+
+This product includes software from the
```

Comment:
ANTLR is not included in the source. Files generated via ANTLR are included in the source. We download, compile and link the ANTLR runtime at build time like all other dependencies.